charter-template.html

```diff
@@ -266,7 +266,7 @@ <h2>Success Criteria</h2>
     </p>
 	
     <!-- Relevance and momentum -->
-	  <p><span class='todo'>Consider adding this clause if the Group does not intend to move to REC:</span> All new features should be supported by at least two intents to implement before being incorporated in the specification.</p>
+	  <p><span class='todo'>Consider adding this clause if the Group does not intend to move to REC:</span> All new features should have expressions of interest from at least two potential implementors before being incorporated in the specification.</p>
```

Comment:
Just double-checking: Do you think "interest" is sufficient, or should this match the WHATWG working mode in requiring ["support"](https://whatwg.org/working-mode#additions)? I think I'll be happier about "interest" once there's also an indication at the CR phase of which features have 2 implementations, and I'm fine with merging this PR in the expectation that that text will arrive later.

Reply:
I'd be fine with the WHATWG wording.